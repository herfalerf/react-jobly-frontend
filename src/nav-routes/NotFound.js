import React from "react";

//Basic dumb component for when user tries to access a route that doesn't exist.
function NotFound() {
  return (
    <div>
      <h1>The page you're looking for doesn't exist</h1>
    </div>
  );
}

export default NotFound;
