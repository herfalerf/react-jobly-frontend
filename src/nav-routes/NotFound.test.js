import React from "react";
import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router";
import NotFound from "./NotFound";
import { UserProvider } from "../testUtils";

it("renders without crashing", function () {
  render(
    <MemoryRouter>
      <UserProvider>
        <NotFound />
      </UserProvider>
    </MemoryRouter>
  );
});

it("matches snapshot", function () {
  const { asFragment } = render(
    <MemoryRouter>
      <UserProvider>
        <NotFound />
      </UserProvider>
    </MemoryRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});
