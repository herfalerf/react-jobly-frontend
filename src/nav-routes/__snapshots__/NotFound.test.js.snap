// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1>
      The page you're looking for doesn't exist
    </h1>
  </div>
</DocumentFragment>
`;
